rpc_jobs/dep_update.yml

```diff
@@ -11,15 +11,25 @@
           URL: "https://github.com/rcbops/rpc-openstack"
           BRANCH: "newton"
           JIRA_PROJECT_KEY: "RO"
+    trigger:
+      - PR:
+          CRON: ""
+          branches: "{BRANCH}"
+          NUM_TO_KEEP: 10
+      - PM:
+          branches: "do_not_build_on_pr"
+          READ_ONLY_TEST: false
     jobs:
-      - 'Dep-Update_{repo}-{BRANCH}'
+      - 'Dep-Update_{repo}-{BRANCH}-{trigger}'
 
 
 - job-template:
     CRON: "@weekly"
     RPC_GATING_BRANCH: master
     JIRA_ISSUE_TYPE: Task
-    name: 'Dep-Update_{repo}-{BRANCH}'
+    NUM_TO_KEEP: 30
+    READ_ONLY_TEST: true
+    name: 'Dep-Update_{repo}-{BRANCH}-{trigger}'
```

Comment:
Should we call this `{trigger}_Dep-Update-{repo}-{branch}` to keep it a bit more standard with other job names?

Reply:
I wanted to, but then the job name linter failed and I didn't want to go down the road of trying to figure out the complex regex.